File: src/main/java/tictactoe/CompletedGamesReport.java
package tictactoe;

public class CompletedGamesReport {
    private int numberOfXWinner;
    private int numberOfOWinner;
    private int numberOfDraw;

    public CompletedGamesReport(int numberOfXWinner, int numberOfOWinner, int numberOfDraw) {
        this.numberOfXWinner = numberOfXWinner;
        this.numberOfOWinner = numberOfOWinner;
        this.numberOfDraw = numberOfDraw;
    }

    public String getContent() {
        return "Winner Summary \n" +
                "==============\n" +
                "Player X: " + numberOfXWinner + "\n" +
                "Player O: " + numberOfOWinner + "\n" +
                "Draw: " + numberOfDraw + "\n";
    }
}

File: src/main/kotlin/tictactoe/CompletedGamesReport.kt
package tictactoe

/**
 * Completed Games Report
 */
data class CompletedGamesReport(
    val numberOfXWinner: Int,
    val numberOfOWinner: Int,
    val numberOfDraw: Int
) {
    fun getContent() = """
        Winner Summary 
        ==============
        Player X: $numberOfXWinner 
        Player O: $numberOfOWinner 
        Draw: $numberOfDraw 
    """.trimIndent()
}